src/ariel-os-coap/src/lib.rs

```diff
@@ -79,9 +79,7 @@ pub async fn coap_run(handler: impl coap_handler::Handler + coap_handler::Report
 
     info!("Starting up CoAP server");
 
-    // Can't that even bind to the Any address??
-    // let local_any = "0.0.0.0:5683".parse().unwrap(); // shame
-    let local_any = "10.42.0.61:5683".parse().unwrap(); // shame
+    let local_any = "0.0.0.0:5683".parse().unwrap(); // shame
```

Comment:
```suggestion
    let local_any = "0.0.0.0:5683".parse().unwrap();
```

Reply:
These are my reminders for every place where we have IPv4 addresses around. Happy to adjust them to something else, but we should have some reminders to grep for them once we can get rid of them.

In this concrete case, I *think* that due to how smoltcp works, we can actually switch already. Adding and testing; in the meantime: any suggestions for a better name for where there are still any around? (The CoAP client examples are probably the only ones left)